packages/mobile/src/recipients/reducer.ts

```diff
@@ -12,44 +12,54 @@ export interface State {
   // valoraRecipientCache contains accounts that the user has sent/recieved transactions from,
   // and includes CIP8 profile data if available
   valoraRecipientCache: AddressToRecipient
+  rewardsSenders: string[]
```

Comment:
More correct if there's no migration:
```suggestion
  rewardsSenders: string[] | undefined
```

Reply:
I added a spread of the `initialState` to the `rehydrate` action so that this isn't necessary, otherwise we'd need to add null checks in different places.